www/src/config.ts

```diff
@@ -387,3 +387,5 @@ export const SIDEBAR_HEADER_MAP: Record<
     Deployment: "部署",
   },
 };
+
+export const RTL_LANGS: KnownLanguageCode[] = ["ar"];
```

Comment:
we already have this https://github.com/t3-oss/create-t3-app/blob/next/www/src/languages.ts#L13